hw9/my_exception.py

```diff
@@ -0,0 +1,25 @@
+from _datetime import datetime
+
+
+class MyException:
```

Comment:
You are confused a context manager and a custom exception. Your custom exception should look like this:
```
class MyCustomException(Exception):
    def __init__(self, message):
        # write to a file here